lib/helpers/ap-service.ts

```diff
@@ -8,12 +9,25 @@ export const getApServiceSettings = (serviceType: TServiceType) => {
   return serviceSettings[serviceType as TServiceType] ?? null
 }
 
-export const getApServiceUrl = (serviceType: TServiceType) => {
+export const getApServiceUrl = async (serviceType: TServiceType) => {
   const serviceSettings = getApServiceSettings(serviceType)
   if (!serviceSettings) {
     throw new Error(`No url found for the ${serviceType} service`)
   }
 
+  if (serviceType === "fbi") {
+    const dplCmsConfig = await getDplCmsPublicConfig()
+
+    if (!dplCmsConfig.searchProfiles.local) {
+      throw new Error("DPL CMS searchProfiles.local is not defined")
+    }
```

Comment:
Are we sure we want to fail hard here or just log it? I am just asking because I am not sure :)